.github/workflows/integration_tests.yml

```diff
@@ -739,11 +739,26 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ env.pythonVersion }}
+      - name: Set up Node (12)
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+      - name: Setup Firestore Emulator
+        if: ${{ !startsWith(matrix.os, 'windows') }}
+        run: |
+          npm install -g firebase-tools
+          firebase emulators:start --only firestore --project demo-example &
       - name: Setup integration test deps
         run: |
           pip install -r scripts/gha/requirements.txt
           python scripts/gha/restore_secrets.py --passphrase "${{ secrets.TEST_SECRET }}" --artifact testapps
-      - name: Run Desktop integration tests
+      - name: Run Desktop integration tests (Non-windows)
+        if: ${{ !startsWith(matrix.os, 'windows') }}
+        run: python scripts/gha/desktop_tester.py --testapp_dir testapps --logfile_name "desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}"
+        env:
+          USE_FIRESTORE_EMULATOR: true
```

Comment:
I don't know all the detail. But I was wondering why Non-windows need this env. 
Isn't `intent.putExtra("USE_FIRESTORE_EMULATOR", "true")` enough to set the env?

Reply:
It is enough for Android. This is running desktop tests.

Reply:
My bad. Then why Windows doesn't have this env?